packages/devtools_app/lib/src/inspector/inspector_screen.dart

```diff
@@ -125,19 +130,19 @@ class InspectorScreenBodyState extends State<InspectorScreenBody>
     addAutoDisposeListener(searchFieldFocusNode, () {
       // Close the search once focus is lost and following conditions are met:
       //  1. Search string is empty.
-      //  2. searchPreventCloseOnBlur = false (this is set true when searchTargetType Dropdown is opened).
+      //  2. searchPreventClose == false (this is set true when searchTargetType Dropdown is opened).
       if (!searchFieldFocusNode.hasFocus &&
           summaryTreeController.search.isEmpty &&
-          !searchPreventCloseOnBlur) {
+          !searchPreventClose) {
         setState(() {
           searchVisible = false;
         });
       }
 
-      // Reset searchPreventCloseOnBlur state to false after the search field gains focus.
+      // Reset searchPreventClose state to false after the search field gains focus.
```

Comment:
dart-doc nit: wrap var names that are in comments in brackets: [searchPreventClose]